packages/account/src/Components/account-limits/withdrawal-limits-table.tsx

```diff
@@ -0,0 +1,144 @@
+import { ButtonLink, Text } from '@deriv/components';
+import { formatMoney, isMobile } from '@deriv/shared';
+import { Localize, localize } from '@deriv/translations';
+import AccountLimitsTableCell from './account-limits-table-cell';
+import AccountLimitsTableHeader from './account-limits-table-header';
+import React from 'react';
+import { observer, useStore } from '@deriv/stores';
+
+type TWithdrawalLimitsTable = {
+    is_app_settings?: boolean;
+    is_appstore: boolean;
+    num_of_days_limit?: string | number;
+    withdrawal_since_inception_monetary?: string | number;
+    remainder?: string | number;
+};
```

Comment:
why do we set these params as optional?

Reply:
@amina-deriv, all these are option from account-limits, that's why they're optional here as well, after I extracted the markup to a separate component.